MaxText/generate_param_only_checkpoint.py

```diff
@@ -90,6 +90,40 @@ def _read_train_checkpoint(config, checkpoint_manager, mesh):
   return state, state_mesh_notations
 
 
+def _read_load_save_lora_checkpoints(config, mesh):
+  """Read lora checkpoints checkpoint at path defined by load_full_state_path."""
+  # Model and Optimizer definition
+  quant = quantizations.configure_quantization(config)
+  model = Transformer(config, mesh, quant)
+  rng = random.PRNGKey(0)
+  learning_rate_schedule = max_utils.create_learning_rate_schedule(config)
+  tx = optimizers.get_optimizer(config, learning_rate_schedule)
+
+  lora_adapters = max_utils.gcs_list_directories(config.lora_adapters_path)
+  for lora_id in lora_adapters:
+    lora_checkpoint_dir = config.checkpoint_dir + config.lora_dir + lora_id + "/"
```

Comment:
nit: this would be cleaner with an f-string

Reply:
Done.